.bazelrc

```diff
@@ -54,3 +54,15 @@ build --incompatible_no_implicit_file_export
 # build --incompatible_struct_has_no_methods
 # TODO(alexeagle): turn on this flag when dependencies allow
 # build --incompatible_use_platforms_repo_for_constraints
+
+###############################
```

Comment:
this already happened above when we ran
`import %workspace%/common.bazelrc`

I dunno if that's a problem or not